kubejs/server_scripts/base/tags/items/create/crushed_ores.js

```diff
@@ -0,0 +1,9 @@
+ServerEvents.tags('item', (event) => {
+    let props = Object.keys(metal_properties);
```

Comment:
This can be done with regex instead of a loop. Would prefer it.

Reply:
I'm going to be honest, I have no idea how to do this with regex 😅

Reply:
I wasn't paying close enough attention on this one ^_^ Nevermind.